actionview/lib/action_view/digestor.rb

```diff
@@ -71,11 +71,16 @@ def tree(name, finder, partial = false, seen = {})
 
       private
         def find_template(finder, *args)
+          name = args.first
+          prefixes = args[1] || []
+          partial = args[2] || false
+          keys = args[3] || []
+          options = args[4] || {}
```

Comment:
Uh, please! Why not just `def find_template(finder, name, prefixes = [], partial = false, keys = [], options = {})`?

@rafaelfranca will you accept PR with this change?

Reply:
even `find_template(finder, *args, **options)` should be fine.